pom.xml

```diff
@@ -22,38 +22,37 @@
     <modelVersion>4.0.0</modelVersion>
     <parent>
         <groupId>org.apache.sling</groupId>
-        <artifactId>sling</artifactId>
-        <version>16</version>
-        <relativePath/>
+        <artifactId>sling-bundle-parent</artifactId>
+        <version>47</version>
+        <relativePath />
     </parent>
 
     <artifactId>org.apache.sling.commons.cache.api</artifactId>
     <version>0.1-SNAPSHOT</version>
-    <packaging>bundle</packaging>
 
     <name>Apache Sling Commons Cache API</name>
     <description>
-        This bundle porvides a Cache API for Sling applications to use.
+        This bundle provides a Cache API for Sling applications to use.
     </description>
 
+    <properties>
+        <sling.java.version>8</sling.java.version>
+        <project.build.outputTimestamp>1</project.build.outputTimestamp>
+        <bnd.baseline.skip>true</bnd.baseline.skip>
```

Comment:
Why is this needed?

Reply:
seems like baseline version was not found from repository. therefore after the next release, we can take out this skip param

[INFO] --- bnd-baseline-maven-plugin:6.1.0:baseline (bnd-baseline) @ org.apache.sling.commons.cache.api ---
[INFO] Determining the baseline version for org.apache.sling:org.apache.sling.commons.cache.api:jar:(,0.1-SNAPSHOT) using repositories [apache.snapshots (https://repository.apache.org/snapshots, default, snapshots), central (https://repo.maven.apache.org/maven2, default, releases)]
[INFO] `The baseline version was found to be null`

`org.apache.sling.commons.cache.api`: An error occurred while calculating the baseline: Unable to locate a previous version of the artifact

Reply:
Fixed